### DISCUSSION TITLE
$hname docs error?

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#72] [Q&A]
[https://github.c om/gitbls/sdm/blob/master/Docs/Programming-Plugins-and-Custom-Phase-Scripts.md](https://github.com/gitbls/sdm/blob/master/Docs/Programming-Plugins-and-Custom-Phase-Scripts.md#what-variables-are-controlled-by-command-line-switches) lists $hname as the variable containing the hostname.  I believe the correct variable name is $hostname.

**Comment by gitbls** [comment]
I believe you're correct.
